### DISCUSSION TITLE
에디터에서 public 변수에 에셋을 등록해도 플레이하면 none으로 바뀝니다.

### DISCUSSION THREAD

**Discussion by HanjuKim786** [naverz/zepeto-studio-kor#842] [5. Q&A]
제목처럼 같은 현상이 발생합니다.
인스펙터에서 에셋을 등록하고 플레이하면 none으로 바뀌면서 null reference  error가 발생합니다.
이러한 현상에 대한 해결 방법이 있을까요?

**Comment by HanjuKim786** [comment]
Saving Prefab to immutable folder is not allowed: Packages/zepeto.character/resources/zepeto/ZepetoBaseModel.prefab
UnityEditor.EditorApplication:Internal_CallGlobalEventHandler ()

에러도 같이 뜨는데 이유를 모르겠습니다.

**Reply by HanjuKim786** [reply]
오브젝트 계층도를 첨부합니다
![재페토 문의](https://user-images.githubusercontent.com/59901263/172256328-71ae6876-9078-40f5-aaca-a3e6f232769a.jpg)
.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요. ZepetoBaseModel.prefab 오류는 known issue이며 https://github.com/naverz/zepeto-studio-kor/issues/789
실행에는 문제가 없습니다.

인스펙터에 등록한 에셋이 none이 되는 경우는 코드에서 직접 추가를 해주거나,
에셋을 프리팹으로 만든 후 project 내 경로에서 연결을 시켜보시기를 추천 드립니다. 

그래도 해결되지 않을 경우 도움을 위해 코드 및 인스펙터 스크린샷과 같이 보다 상세한 정보를 올려주시면 좋겠습니다.
